lib/msf/core/web_services/servlet/host_servlet.rb

```diff
@@ -80,6 +87,53 @@ def self.delete_host
     }
   end
 
+  def self.get_host_tags
+    lambda {
+      warden.authenticate!
+      begin
+        opts = parse_json_request(request, false)
+
+        tmp_params = sanitize_params(params)
+        opts[:id] = tmp_params[:id] if tmp_params[:id]
+        data = get_db.get_host_tags(opts)
+        set_json_data_response(response: data)
+      rescue => e
+        print_error_and_create_response(error: e, message: 'There was an error adding the host tag:', code: 500)
```

Comment:
Minor: error was in "retrieving" not "adding"

Reply:
Good catch, bad 👮‍♂️ 🍝 on my part